File: random-bot.js
const CONSTANTS = require('./helper-package-js/constants');
const { ACTION, FACE, FLOWER, MOVE } = require('./helper-package-js/constants');

const { Action } = require('./helper-package-js/action');
const { Bot } = require('./helper-package-js/bot');
const { Position } = require('./helper-package-js/position');

class RandomBot extends Bot {
    constructor() {
        super("RandomBot");
    }

    getActions() {
        const actions = [];

        for (let beeCell of this.beeCells) {
            const move = Math.floor(Math.random() * 5);
            actions.push(new Action({
                type: ACTION.MOVE,
                pos: beeCell.pos,
                move: move,
                face: move > 0 ? move - 1 : beeCell.bee.face,
            }));
        }

        return actions;
    }
}

(async function main() {
    const randomBot = new RandomBot();
    await randomBot.run();
})();

File: random_bot.py
from helper_package_python.constants import (
	ActionType, 
	Move, 
	Face, 
	Flower, 
	HIVE_POLLEN_AMOUNT, 
	BEE_POLLEN_AMOUNT,
	MAX_BEES,
	NEUTRAL_ID
)

from helper_package_python.bot import Bot
from helper_package_python.action import Action
from helper_package_python.position import Position

from random import randint

class RandomBot(Bot):
	def __init__(self):
		super().__init__("RandomBot")

	def getActions(self):
		actions = []

		for beeCell in self.beeCells:
			move = randint(0, 4)

			actions.append(Action(
				ActionType.MOVE,
				beeCell.pos,
				move,
				(move - 1) if (move > 0) else beeCell.bee.face
			))

		return actions

def main():
	randomBot = RandomBot()
	randomBot.run()

main()
